warden/app/app_config.go

```diff
@@ -203,134 +263,158 @@ var (
 		// We allow the following module accounts to receive funds:
 		// govtypes.ModuleName
 	}
+)
 
+func moduleConfig() depinject.Config {
```

Comment:
i changed this a function instead of a `var` as i needed the `init` function to run first in order to set the bech32 prefixes.